Sources/App/add.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  remote.swift
+//  Guaka
+//
+//  Created by Omar Abdelhafith on 05/11/2016.
+//
+//
+
+import Guaka
+
+var addCommand: Command = Command(
```

Comment:
Just a question. Can't we just omit the type here? Just using the inference. Is it a swift idiom to state types like this? #justaskingbro :)